core/src/Streamly/Internal/Data/Stream/Expand.hs

```diff
@@ -120,32 +120,31 @@ module Streamly.Internal.Data.Stream.Expand
 
     -- * Iterate
     -- | Map and flatten Trees of Streams
-    , iterateUnfoldManyDFS
-    , iterateUnfoldManyBFS
-    , iterateUnfoldManyBFSRev
+    , unfoldIterateDfs
+    , unfoldIterateBfs
+    , unfoldIterateBfsRev
 
-    , iterateConcatMapWith
-    , iterateMergeMapWith
+    , concatIterateWith
+    , mergeIterateWith
 
-    , iterateConcatMapDFS
-    , iterateConcatMapBFS
+    , concatIterateDfs
+    , concatIterateBfs
```

Comment:
Why?

Reply:
small case `s`.

Reply:
Mixing all caps abbreviations with camel case is problematic e.g. unfoldIterateBFSRev. See https://github.com/composewell/haskell-dev/blob/master/coding-style.rst#id8 .